src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -91,14 +91,13 @@ message ValidationFailures {
 }
 
 message SecurityGroupValidationRequest {
-    // A security group value can be:
-    // * a security group id (for example sg-12345678)
-    // * a security group ARN
-    // * a security group name/alias (for example 'titus-sg')
-    string securityGroup = 1;
-
-    // A subnet id associated with the given security group. If not set, a default subnet for the given account is used.
-    string subnetId = 2;
+    // Security group IDs (sg-[0-9a-f]+), for example: sg-12345678
+    repeated string securityGroups = 1;
```

Comment:
Don't reuse numbers. You can drop existing fields, we just need to not reuse numbers, otherwise it'll break on deserialization for old deserializers.